packages/accordions/examples/advanced.md

```diff
@@ -0,0 +1,191 @@
+  }
+];
+
+const AdvancedExample = () => {
+  const [expandedSections, setExpandedSections] = React.useState([0]);
+  const [isBare, setIsBare] = React.useState(false);
+  const [isCompact, setIsCompact] = React.useState(false);
+  const [isExpandAll, setIsExpandAll] = React.useState(false);
+  const [isCollapseAll, setIsCollapseAll] = React.useState(false);
+  const onExpandAll = () => {
+    setIsExpandAll(!isExpandAll);
+    setIsCollapseAll(false);
+    setExpandedSections([0, 1, 2, 3, 4]);
+  };
+  const onCollapseAll = () => {
+    setIsCollapseAll(!isCollapseAll);
+    setIsExpandAll(false);
+    setExpandedSections([]);
+  };
+  const onChange = index => {
+    setExpandedSections(sections => [index]);
+    setIsExpandAll(false);
+    setIsCollapseAll(false);
+  };
+
+  const buttonGroup = (
+    <StyledButtonGroup>
+      <Dropdown onSelect={item => alert(item)}>
+        <Trigger>
+          <IconButton aria-label="Settings" title="Settings">
+            <GearIcon />
+          </IconButton>
+        </Trigger>
+        <Menu placement="top" hasArrow>
+          <Item value="option-1">Option 1</Item>
+          <Item value="option-2">Option 2</Item>
+          <Item value="option-3">Option 3</Item>
+        </Menu>
+      </Dropdown>
+      <Dropdown onSelect={item => alert(item)}>
+        <Trigger>
+          <IconButton aria-label="Settings" title="Settings">
+            <FolderIcon />
+          </IconButton>
+        </Trigger>
+        <Menu placement="top" hasArrow>
+          <Item value="option-1">Option 1</Item>
+          <Item value="option-2">Option 2</Item>
+          <Item value="option-3">Option 3</Item>
+        </Menu>
+      </Dropdown>
+    </StyledButtonGroup>
+  );
+
+  return (
+    <Grid>
+      <Well isRecessed>
+        <Row>
+          <Col>
+            <Field>
+              <Toggle checked={isBare} onChange={event => setIsBare(!isBare)}>
+                <Label>Bare</Label>
+              </Toggle>
+            </Field>
+            <Field className="u-mt">
+              <Toggle checked={isExpandAll} onChange={onExpandAll}>
+                <Label>Expand All</Label>
+              </Toggle>
+            </Field>
+            <Field className="u-mt">
+              <Toggle checked={isCollapseAll} onChange={onCollapseAll}>
+                <Label>Collapse All</Label>
+              </Toggle>
+            </Field>
+          </Col>
+        </Row>
+      </Well>
+      <Row className="u-mt">
+        <Col>
+          <Accordion
+            level={3}
+            isBare={isBare}
+            onChange={onChange}
+            expandedSections={expandedSections}
+          >
+            {accordionSections.map(section => (
+              <Accordion.Section key={section.id}>
+                <Accordion.Header>
+                  <Accordion.Label>
+                    {section.header}
+                    {'\u00A0'}
```

Comment:
I don't think this necessity for a non-breaking space is a practice we can advocate for in implementation of this component. I'm unsure of the best solve but we should space correctly out of the box. Maybe margin-left on the span element?

Reply:
@rossmoody I can use CSS spacing instead, but could you help me understand the consequences of using an HTML entity here?